cmd/bosun/web/static/js/graph.ts

```diff
@@ -466,7 +469,12 @@ bosunControllers.controller('GraphCtrl', ['$scope', '$http', '$location', '$rout
 		});
 		return request;
 	}
-	$scope.Query = function() {
+		$scope.keydown = function ($event: any) {
```

Comment:
If you are over a link and hit shift-enter it opens the link a new tab. But maybe it could globally be applied to text input fields for the view if you can think of a way to do that.